fortnite_api/shop.py

```diff
@@ -123,18 +125,13 @@ def from_value(cls: Type[Self], value: str, /) -> Self:
         -------
         :class:`TileSize`
             The constructed tile size.
-
-        Raises
-        ------
-        ValueError
-            If the value is not in the correct format.
         """
         # Try and match the regex
         match = _TILE_SIZE_REGEX.match(value)
         if not match:
-            # Epic only uses the tile sizing on the regex. If there isn't a match
-            # we can safely assume an exception must be raised
-            raise ValueError(f"Invalid tile size: {value!r}")
+            # Epic may change the format or messes up the value.
+            # In one such case, this value was "ERROR BAD SIZE".
+            raise cls(width=1, height=1, internal=value)
```

Comment:
I choose 1x1 as a fallback value for TileSize. An alternative would be 0x0, hence items with invalid tile sizes aren't rendered

Reply:
I think this should actually be `-1` to indicate a failed parsing. What do you think?

Reply:
`-1` is common as an error indicator, but am not sure. 0x0 would be also a value that shouldn't be returned by the API.